dbms/src/IO/Compression/CompressionInfo.h

```diff
@@ -14,33 +14,18 @@
 
 #pragma once
 
-#include <Common/config.h>
-
 #include <cstdint>
 
-/** Common Defines */
+/** Common defines for compression */
 
 #define DBMS_MAX_COMPRESSED_SIZE 0x40000000ULL /// 1GB
 
+/** one byte for method, 4 bytes for compressed size, 4 bytes for uncompressed size */
 #define COMPRESSED_BLOCK_HEADER_SIZE 9
 
-
 namespace DB
 {
-/** Compression method */
-enum class CompressionMethod
-{
-    LZ4 = 1,
-    LZ4HC = 2, /// The format is the same as for LZ4. The difference is only in compression.
-    ZSTD = 3, /// Experimental algorithm: https://github.com/Cyan4973/zstd
-#if USE_QPL
-    QPL = 4, /// The Intel Query Processing Library (QPL) is an open-source library to provide high-performance query processing operations
-    NONE = 5, /// No compression
-#else
-    NONE = 4, /// No compression
-#endif
-};
-
+using UInt8 = std::uint8_t;
```

Comment:
Use `#include <common/types.h>` to avoid defining another `DB::UInt8` rather thant `::UInt8`?

Reply:
done